ashkan-rahnavardi/AshkansWorld
# Update text scramble text component to a hook, remove unused imports, switch to yarn as package manager

Changes are mostly a refactor of the text scrambling feature, aiming to enhance maintainability and performance. 

Major changes include:

**Removal of External Library**: 
phased out the @twistezo/react-text-scramble library and its dependent components (example.tsx and scramble.tsx), 

**Custom Hook Implementation**: 
The feature now utilizes a custom React hook, useTextScramble. This hook is integrated into start.tsx, due to issues with the old scramble package not playing nicely with next.js. 

**Main Page**: 
Corresponding to the refactor, the page.tsx has been simplified by removing the now-obsolete Example component.

**Switch to yarn over npm** 
Forcing yarn on this repo because it's better than npm


**To test changes**

- run `yarn` to create the new yan.lock file
- `yarn dev` to start the project
- The landing page should appear the same as before if all worked well
